clang/test/dpct/asm/cp.cu

```diff
@@ -67,4 +66,34 @@ __device__ inline void cp_async4_pred(void *smem_ptr, const void *glob_ptr,
                "r"(smem), "l"(glob_ptr), "n"(BYTES));
 }
 
+// CHECK:inline void cp_async_commit_group() {
+// CHECK-NEXT:  /*
+// CHECK-NEXT:  DPCT1026:{{[0-9]+}}: The call to "cp.async.commit_group;" was removed because synchronous copy operation is used. You may need to adjust the code to tune the performance.
```

Comment:
```suggestion
// CHECK-NEXT:  DPCT1026:{{[0-9]+}}: The call to "cp.async.commit_group;" was removed because current cp.async is migrated to synchronous copy operation. You may need to adjust the code to tune the performance.
```

Reply:
Updated